src/views/About.jsx

```diff
@@ -0,0 +1,89 @@
+import './About.css';
+
+export function About() {
+	return (
+		<>
+			<h1>About</h1>
+			<article className="about-article">
+				<h2 className="article-title">What is Shop-Ade</h2>
+				<p>
+					<b>Shop-Ade</b> is a smart shopping list that learns your buying
+					habits and helps you remember what you need to buy on your next trip
+					to the store. Each time you buy the item, you mark it as purchased in
+					the list. Over time, the app comes to understand the intervals at
+					which you buy different items. If an item is likely to be due to be
+					bought soon, it rises to the top of the shopping list.
+				</p>
+			</article>
+			<article className="about-article">
+				<h2 className="article-title">How to use Shop-Ade</h2>
+
+				<ul>
+					<li>
+						To get started with your first list - click Create button in the
+						Home screen.
+					</li>
+					<li>
+						To join an already existing list - enter the list name in the first
+						field.
+					</li>
+					<li>
+						If you want to add an item to your list - click on Add Item button,
+						enter the item name, select the time frame when you want to buy this
+						item out of three options - "soon", which means within 7 days, "kind
+						of soon" which means within 8-14 days, "not so soon", which means
+						within 15-30 days.
+					</li>
+					<li>
+						If made a mistake when entering the item name - you can re-enter the
+						name by selecting Submit button. Click Submit when you finish
+						correcting the item name. The name will be immediately updated.
+					</li>
+					<li>
+						If you want to delete an item from your list - click X button near
+						the item name. You will see a confirmation dialog, if you select OK
+						in the dialog window the item will be immediately deleted.
+					</li>
+					<li>
+						If you bought an item - mark it as purchased in the list by putting
+						a checkmark. The item will uncheck itself after 24 hours. The app
+						will use the purchase information to predict next purchase and will
+						move the item to one of the four time frames.
+					</li>
+					<li>
```

Comment:
I think the content here is good -- potentially not needed but if you do want to keep it I would recommend nesting the content under expandable sections so that you can skip over the sections you don't need info on quickly

like 

<details><summary>Adding to your list   </summary>If you want to add an item to your list - click on Add Item button,
						enter the item name, select the time frame when you want to buy this
						item out of three options - "soon", which means within 7 days, "kind
						of soon" which means within 8-14 days, "not so soon", which means
						within 15-30 days.</details>
<details><summary>Editing the list   </summary>If made a mistake when entering the item name - you can re-enter the
						name by selecting Submit button. Click Submit when you finish
						correcting the item name. The name will be immediately updated.</details>
<details><summary>Deleting items from the list   </summary> 
						If you want to delete an item from your list - click X button near
						the item name. You will see a confirmation dialog, if you select OK
						in the dialog window the item will be immediately deleted.</details>
<details><summary> Updating purchases  </summary>If you bought an item - mark it as purchased in the list by putting
						a checkmark. The item will uncheck itself after 24 hours. The app
						will use the purchase information to predict next purchase and will
						move the item to one of the four time frames.</details>

Reply:
This might be a personal preference but if it interests you to have these expandable sections  the markup for this is using the details and summary html elements like this:

 `<details><summary>{Brief Description}</summary>{Long Description} </details>`

Reply:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/details

Reply:
So many HTML native elements I am to discover! Thank you @lindseyindev 
I really like the idea of expandable sections

Reply:
I actually really like this idea too. I think it would be good to have a brief description of the app, but then have these expandable sections for more detailed instructions. I think it gives some more life to the app!